packages/broker/src/plugins/operator/MaintainTopologyService.ts

```diff
@@ -1,68 +1,110 @@
-import { Logger, Multimap } from '@streamr/utils'
-import { MaintainTopologyHelper } from './MaintainTopologyHelper'
-import StreamrClient, { Stream, Subscription } from 'streamr-client'
-import { StreamID, StreamPartIDUtils } from '@streamr/protocol'
+import { Logger } from '@streamr/utils'
+import StreamrClient, { Subscription } from 'streamr-client'
+import { StreamPartID, StreamPartIDUtils, toStreamID } from '@streamr/protocol'
 import pLimit from 'p-limit'
+import { StreamAssignmentLoadBalancer } from './StreamAssignmentLoadBalancer'
+import { MaintainTopologyHelper } from './MaintainTopologyHelper'
+import { OperatorServiceConfig } from './OperatorPlugin'
+import { OperatorFleetState } from './OperatorFleetState'
 
 const logger = new Logger(module)
 
+/**
+ * Helper function for setting up and starting a MaintainTopologyService along
+ * with all its dependencies.
+ */
+export async function setUpAndStartMaintainTopologyService({
```

Comment:
Is this a helper method `MaintainTopologyService.test.ts` as this is not called from anywhere else? Could move there?

Reply:
The idea would be that `setUpAndStartMaintainTopologyService` would also be used in source code. Specifically from [packages/broker/src/plugins/operator/OperatorPlugin.ts](https://github.com/streamr-dev/network/pull/1643/files/08246eadfa7e1ec3aad8137765e57ce979f252c8#diff-c73edeaf0a46f53e011c4cd619ea4e2e3fc7c13c5c7ae571b0899c6153f3be4a) here.